config/base.json

```diff
@@ -14,6 +14,7 @@
     "max_static_paths": 10
   },
   "flags": {
-    "enable_new_downloads_view": true
+    "enable_new_downloads_view": true,
+    "enable_product_docs_search": true
```

Comment:
New flag. Disabled in "production" for now, but it handles showing the new search element and hiding the placeholder in the header.